pkgcfg/pkgcfg.go

```diff
@@ -77,7 +80,29 @@ func get(pkg, rootDir string) (*config, error) {
 		debug.Log("Using local package config at %q", configPath)
 		return getLocalConfig(configPath, pkg, rootDir)
 	}
-	return &config{}, nil
+	return getConfig(pkg, rootDir)
+}
+
+var baseConfigURL = "https://raw.githubusercontent.com/jetpack-io/devbox/main/pkgcfg/package-configuration/dummy-config"
+
+func getConfig(pkg, rootDir string) (*config, error) {
+	confURL, err := url.JoinPath(baseConfigURL, pkg+".json")
```

Comment:
Wait so everything lives under `/package-configuration/dummy-config/`? Why dummy-config?

Reply:
No, it lives under `package-configuration` I just need to test with exciting directory because https://github.com/jetpack-io/devbox/pull/263 hasn't landed. I'll update.